### ISSUE TITLE
When UglifyJS fails the errors are not reported by UglifyJSPlugin

### ISSUE THREAD

**Issue by xaviergonz** [fuse-box/fuse-box#724]
```
TypeError: "string" must be a string, Buffer, or ArrayBuffer
    at Function.byteLength (buffer.js:477:11)
    at UglifyJSPluginClass.postBundle (C:\VSProjects\gitlab\skirmish-vt-4\webclient\node_modules\fuse-box\dist\commonjs\plugins\UglifyJSPlugin.js:29:28)
    at plugins.forEach.plugin (C:\VSProjects\gitlab\skirmish-vt-4\webclient\node_modules\fuse-box\dist\commonjs\core\WorkflowContext.js:383:34)
    at Array.forEach (native)
    at WorkFlowContext.triggerPluginsMethodOnce (C:\VSProjects\gitlab\skirmish-vt-4\webclient\node_modules\fuse-box\dist\commonjs\core\WorkflowContext.js:368:22)
    at FuseBox.triggerPost (C:\VSProjects\gitlab\skirmish-vt-4\webclient\node_modules\fuse-box\dist\commonjs\core\FuseBox.js:124:22)
    at realm_utils_1.chain.then.then.result (C:\VSProjects\gitlab\skirmish-vt-4\webclient\node_modules\fuse-box\dist\commonjs\core\FuseBox.js:270:26)
    at <anonymous>
```

```js
const production = true;
const fuse = FuseBox.init({
  homeDir: 'src',
  output: `dist/$name.bundle.js`,
  plugins: [
    fb.EnvPlugin({ NODE_ENV: production ? "production" : "development" }),
    fb.TypeScriptHelpers(),
    fb.JSONPlugin(),
    production && fb.UglifyJSPlugin()
  ],
  // hash: production,
  cache: !production,
  log: false,
  debug: false,
  sourceMaps: production ? false : { project: true, vendor: false }
});
```

**Comment by nchanged** [comment]
Make sure you have `THE` latest [uglify-js](https://github.com/mishoo/UglifyJS2/blob/master/package.json#L7) installed. They made a breaking change.

**Comment by nchanged** [comment]
And I would recommend switching to Quantum for production, you could remove UglifyJS from the plugin list and just use `uglify: true`

**Comment by xaviergonz** [comment]
I'm using uglify-js 3.0.27, the latest version according to npm. is there a newer one?
Btw, thanks for the quantum tip

**Comment by Shepless** [comment]
@devmondo @xaviergonz this was resolved in the VuePlugin PR #832 [here](https://github.com/fuse-box/fuse-box/pull/832/files#diff-e1a51f1b91758ca57b41d2ec4a0f517cR63)

I think this issue can be closed now.
